downstream/modules/platform/con-hs-eda-controller.adoc

```diff
@@ -0,0 +1,23 @@
+[id="con-hs-eda-controller"]
+
+= Horizontal scaling in {EDAcontroller} 
+
+With {EDAcontroller}, you can set up horizontal scaling for your events automation. This multi-node deployment enables you to define as many nodes as you prefer during the installation process. You can also increase or decrease the number of nodes at any time according to your organizational needs.
+
+The following node types are used in this deployment:
+
+API node type:: Responds to the HTTP REST API of {EDAcontroller}.
+Worker node type:: Runs an Event-Driven Ansible worker, which is the component of Event-Driven Ansible that not only manages projects and activations, but also executes the activations themselves.
```

Comment:
"Event-Driven Ansible" should be replaced by the attribute {EDAName} in this sentence.